docs/guide/environmental_variables.rst

```diff
@@ -0,0 +1,71 @@
+Environmental Variables
```

Comment:
I'm not native english speaker, but isn't it «environment variables» instead of «environmental»?